lib/bootloader_setup.pm

```diff
@@ -88,14 +88,16 @@ sub bootmenu_default_params {
     if (check_var('VIRSH_VMM_FAMILY', 'hyperv')) {
         type_string " video=hyperv_fb:1024x768";
     }
+    return if is_jeos || is_casp('VMX');
```

Comment:
Wouldn't it make more sense to have this below line 93? Otherwise you're going to skip the Y2DEBUG, which you probably still want